packages/itwinui-react/src/core/Tile/Tile.test.tsx

```diff
@@ -273,45 +273,121 @@ it('should propagate misc props correctly', () => {
 
 it('should render actionable tile', () => {
   const { container } = render(
-    <Tile>
+    <Tile.Wrapper>
       <Tile.Name>
         <Tile.NameLabel>
           <Tile.Action>test-name</Tile.Action>
         </Tile.NameLabel>
       </Tile.Name>
-    </Tile>,
+    </Tile.Wrapper>,
   );
   expect(container.querySelector('.iui-tile.iui-actionable')).toBeTruthy();
 });
 
 it.each(['positive', 'warning', 'negative'] as const)(
-  'should render tile with %s status',
+  'should render Tile with %s status',
   (status) => {
     const { container } = render(
-      <Tile status={status}>
+      <Tile.Wrapper status={status}>
         <Tile.Name name='test-name' />
-      </Tile>,
+      </Tile.Wrapper>,
     );
     expect(container.querySelector(`.iui-tile.iui-${status}`)).toBeTruthy();
   },
 );
 
-it('should render tile with loading status', () => {
+it('should render Tile with loading status', () => {
   const { container } = render(
-    <Tile isLoading={true}>
+    <Tile.Wrapper isLoading={true}>
       <Tile.Name name='test-name' />
-    </Tile>,
+    </Tile.Wrapper>,
   );
   expect(container.querySelector(`.iui-tile.iui-loading`)).toBeTruthy();
 });
 
-it('should render tile with disabled status', () => {
+it('should render Tile with disabled status', () => {
   const { container } = render(
-    <Tile isDisabled={true}>
+    <Tile.Wrapper isDisabled={true}>
       <Tile.Name name='test-name' />
-    </Tile>,
+    </Tile.Wrapper>,
   );
   const tile = container.querySelector(`.iui-tile`) as HTMLElement;
   expect(tile).toBeTruthy();
   expect(tile).toHaveAttribute('aria-disabled', 'true');
 });
+
+it('should support Tile with legacy props', () => {
+  render(
+    <Tile
+      data-testid='1'
+      name='Tile name'
+      description='Tile description that takes upto 3 lines'
+      metadata={'basic metadata'}
+      thumbnail='/url/to/image.jpg'
+      badge={<Badge backgroundColor='blue'>Badge label</Badge>}
+      buttons={[
+        <Button key='button-1'>Button 1</Button>,
+        <Button key='button-2'>Button 2</Button>,
+      ]}
+      moreOptions={[
+        <MenuItem key='menuitem-1'>Item 1</MenuItem>,
+        <MenuItem key='menuitem-1'>Item 2</MenuItem>,
+      ]}
+      leftIcon={
+        <Tile.IconButton>
```

Comment:
why `IconButton` doesn't work here?

Reply:
It works. Users must pass size='small' button for it to be "as designed" 
left is Tile.IconButton, right is IconButton:
<img width="315" alt="image" src="https://github.com/iTwin/iTwinUI/assets/83585998/c9c6ea00-bdf6-4cbd-ae7f-985a39d97ef6">

+ off topic:
I think we should fix dark icons within those buttons. CSS tests have them white.

Reply:
oh wow, the icons are messed up only in light theme so i didn't even notice 😅
<img width="346" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/4aaf9754-3730-44bf-aa9d-dfd118a37a5f">

Reply:
for now i would say use `IconButton` with manual `size='small'` in this unit test. we don't want to mix and match old and new apis.

the color issue can be fixed in a separate PR